README.md

```diff
@@ -79,15 +79,25 @@ If you click on the icon which looks like a set of gears, you should see an inte
 
 ![Rhasspy settings](rhasspy/images/Rhasspy_settings.PNG)
 
-It is recommended that you set all of the settings to the same thing as pictured above, but you may use whichever text to speech option you like. You will also have to fill out the device information underneath the Audio Recording and Audio Playing settings. You can also choose whichever wake word you prefer under Wake Word. When you're done, click on the Save Settings button. Please note that you may have to do this iteratively, and you may be asked to download data for some of the settings. You can test the audio recording and playback by clicking on the icon that looks like a house.
+It is recommended that you set all of the settings to the same thing as pictured above, but you may use whichever text to speech option you like. You can also choose whichever wake word you prefer under Wake Word. When you're done, click on the Save Settings button. Please note that you may have to do this iteratively, and you may be asked to download data for some of the settings. A download button should appear at the top of the page like below if aditional data is needed.
+
+![image](https://github.com/wyattschool/sandman/assets/77877259/505852bd-9a8d-4181-b9b4-bb2e95cb26bb)
```

Comment:
While still learning about these code reviews, is there a way I can see these image assets? I usually put at least a little bit descriptive text inside of the brackets, this would be used by people with low vision or screen readers. It's also not clear where the images are going to be put. We do have a directory for images under documentation.

Reply:
I agree with your feedback. The images to be added should go in the documentation/images directory and more helpful alt text should be added for each photo.